omnipaxos/src/ballot_leader_election.rs

```diff
@@ -75,12 +75,14 @@ pub(crate) struct BallotLeaderElection {
     /// The current round of the heartbeat cycle.
     hb_round: u32,
     /// Vector which holds all the received heartbeats
-    ballots: Vec<(Ballot, Connectivity)>,
+    ballots: Vec<(Ballot, Connectivity, bool)>,
```

Comment:
For code clarity, I think it would be nicer to, instead of using bool, have an alias for `Happy`  or perhaps even better have a struct 
```
BallotResponse {
      ballot: Ballot,
      connectivity: u8 // or Connectivity
      happy: bool
}
```